evm/src/main/java/org/hyperledger/besu/evm/precompile/AltBN128AddPrecompiledContract.java

```diff
@@ -26,14 +25,15 @@
 
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.MutableBytes;
+import org.jetbrains.annotations.NotNull;
```

Comment:
not sure how we feel about putting IDE specific annotations in the code. Was this supposed to be removed after the resulting warnings were resolved?

Reply:
I thought it would code-complete to the javax version when I alt-entered.  Will fix.